distributed/bokeh/worker_monitor.py

```diff
@@ -68,8 +68,11 @@ def resource_append(lists, msg):
     L = list(msg.values())
     if not L:
         return
-    for k in ['cpu', 'memory-percent']:
-        lists[k].append(mean(pluck(k, L)) / 100)
+    try:
+        for k in ['cpu', 'memory-percent']:
+            lists[k].append(mean(pluck(k, L)) / 100)
+    except KeyError:
+        return
```

Comment:
Maybe add an inline comment to explain why this can happen and can be safely ignored?